packages/manager/modules/netapp/src/dashboard/vrack-configuration/controller.js

```diff
@@ -0,0 +1,57 @@
+import { LABELS } from './constants';
+
+export default class OvhManagerNetAppVrackConfigurationCtrl {
+  /* @ngInject */
+  constructor($translate, Alerter, NetappVrackConfigurationService) {
+    this.$translate = $translate;
+    this.Alerter = Alerter;
+    this.NetappVrackConfigurationService = NetappVrackConfigurationService;
+    this.LABELS = LABELS;
+  }
+
+  $onInit() {
+    if (!this.availableVracks?.length) {
+      const noVrack = {
+        internalName: this.$translate.instant(
+          'netapp_vrack_configuration_no_vrack_field',
+        ),
+      };
+      this.availableVracks.push(noVrack);
```

Comment:
What about if `this.availableVracks` is null ?

Reply:
It can't be null as it is a filter on an array so we are sure it is an array